packages/icons/package.json

```diff
@@ -13,8 +13,7 @@
     "test:cov": "echo no test",
     "test:demo": "node ./scripts/docs.js",
     "start": "echo nothing to start",
-    "svgo": "svgo -f src/svg --config=.svgo-icons.yml && svgo -f src/filters --config=.svgo-filters.yml",
-    "prepare-jar": "mvn package"
```

Comment:
Why we loose `prepare-jar` script, not used anymore?

Reply:
Just to understand

Reply:
yes, it was another bundle we did before to publish a jar containing the icons too for components team. But it's not done anymore

Reply:
Ok, thanks for that! Just approved